### PR TITLE
Fix: Plugin registrant warning

### PR DESCRIPTION
**Pull request by ueman**
## :scroll: Description
I moved the class `SentryFlutterWeb` from `src/sentry_flutter_web.dart` to `sentry_flutter_web.dart`.
I also enabled [this](https://dart-lang.github.io/linter/lints/implementation_imports.html) lint to prevent this kind of errors in
the future.

I'm not sure if this is considered a breaking change or not. Technically it is, because the imports changed I guess.

## :bulb: Motivation and Context
See #314

## :green_heart: How did you test it?
I've enabled the lint and it didn't show any warnings.

## :pencil: Checklist

- [x] I reviewed submitted code
- [ ] I added tests to verify changes
- [ ] I updated the docs if needed
- [x] All tests passing
- [ ] No breaking changes

### CODE REVIEW

File: flutter/analysis_options.yaml [hunk 1]
```diff
@@ -24,3 +24,4 @@ linter:
   rules:
     prefer_relative_imports: true
     unnecessary_brace_in_string_interps: true
+    implementation_imports: true
```
Comment (marandaneto) [inline flutter/analysis_options.yaml:27]: currently, we're using `package:pedantic`, is there a more aggressive solution? often I see us enabling new rules that should be avoided in the 1st place :D (not for this PR though)

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,7 @@
 * Fix: Multiple FlutterError.onError calls in FlutterErrorIntegration (#345)
 * Fix: Pass hint to EventProcessors (#356)
 * Fix: EventProcessors were not dropping events when returning null (#353)
+* Fix: Plugin Registrant class moved to barrel file (#314)
```
Comment (marandaneto) [inline CHANGELOG.md]: by Danger, requires PRs id

**Comment by marandaneto** [comment]
> I'm not sure if this is considered a breaking change or not. Technically it is, because the imports changed I guess.

theoretically, no as the import as code generated right, but technically yes, mmm, we could take the same approach as https://github.com/getsentry/sentry-dart/pull/357/files#r594101972
adding a break changes section and explaining what to do shortly.
we are in a pre-release stage as well, so it'd be fine.

File: flutter/analysis_options.yaml (hunk 1 above)
Comment (ueman) [reply flutter/analysis_options.yaml:27]: As per https://dart-lang.github.io/linter/lints/ there are three official rule sets: pedantic, effective_dart and flutter. 
There's also https://pub.dev/packages/lint which describes itself as very strict.

**Review by marandaneto — APPROVED** [review]
LGTM

File: flutter/analysis_options.yaml (hunk 1 above)
Comment (marandaneto) [reply flutter/analysis_options.yaml:27]: thanks, I'll have a look at this one (lint) later this week.

**Pull request merged**